docs/wifi-menu.8.txt

```diff
@@ -0,0 +1,89 @@
+WIFI-MENU(8)
+============
+
+NAME
+----
+wifi-menu - Interactively connect to a wireless network
+
+
+SYNOPSIS
+--------
+*wifi-menu* [-o | --obscure] [+INTERFACE+]
+
+*wifi-menu* [-h | --help]
+
+
+DESCRIPTION
+-----------
+*wifi-menu* allows a user to interactively connect to a wireless network
+over INTERFACE using an existing netctl profile, or an automatically
+generated profile for basic configurations. If only one wireless
+interface is available, INTERFACE can be omitted.  If no interface is
+specified and more than one is detected, a choice of interfaces is
+offered.
+
+*wifi-menu* is only able to generate netctl profiles for simple network
+configurations. Profiles for more complex configurations must be set up
+manually, or through the use of other tools.
+
+OPTIONS
+-------
+
+*-h, --help*::
+    Print a short help text and exit.
+
+*-o, --obscure*::
+    Do not echo password, and store derived pre-shared key
+    in profile instead of plaintext password.
+
++INTERFACE+::
+    The wireless network interface to use for connecting.
+
+
+EXIT STATUS
+-----------
+
+*0*::
+    The interface connected to the wireless network successfully.
+
+*1*::
+    The connection attempt was cancelled.
+
+*2*::
+    The connection attempt failed.
+
+*3*::
+    No wireless networks were found.
+
+*4*::
+    Invalid passphrase length (WEP keys must be between 8 and 63 characters
+    in length).
+
+*7*::
+    An unexpected error code was received.
+
+*255*::
+    The connection attempt was aborted (or an error occurred).
+
+
+NOTES
+-----
+The program may display a black screen for up to a minute when starting a
+connection.
```

Comment:
I believe this one got lost somewhere. If you disagree with my preference to make 'minute' refer less to a specific unit of time, I'd like to hear why.
```suggestion
Establishing a connection may take a minute. While establishing a connection,
nothing is shown on screen.
```